cdap-docs/developers-manual/source/advanced/class-loading.rst

```diff
@@ -0,0 +1,140 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _class-loading:
+
+=============================================================
+Java Class Loading and Distributed Data Processing Frameworks
+=============================================================
+
+Java class loading is one of the most fundamental and powerful concepts provided by the
```

Comment:
Should we remove this first paragraph? Also replace the heading on line 8 with 'Overview/Introduction'? I think this paragraph is excellent for a blog, but less suitable for product documentation?